src/brewscaleini.h

```diff
@@ -49,7 +49,13 @@ unsigned long previousMillistempanalogreading;      // ms for analogreading
     float weightBrew = 0;                               // weight value of brew
     float scaleDelayValue = 2.5;                        // value in gramm that takes still flows onto the scale after brew is stopped
     bool scaleFailure = false;
-    const unsigned long intervalWeight = 200;           // weight scale
+    const unsigned long intervalWeight = 50;           // weight scale
```

Comment:
Increased the scale frequency -- why was it so low before? 
Potentially there could be a further change to only read the scale during shots to save some CPU cycles when idling?